katas/main/src/pokerhands/juan/PokerHands.cs

```diff
@@ -0,0 +1,155 @@
+﻿using HandJuan;
+using HandtypeCheckerJuan;
+using PlayedHandJuan;
+
+namespace PokerHandsJuan
+{
+    public class PlayHands
```

Comment:
This is also great, has a good separation of methods to concentrate on each behavior we need. It can be improved by using more object methods rather that class methods (static). The benefit of using object methods is that we can work with OOP features like inheritance that allow us to expand functionality without making a single class to grow every time we need more features

Reply:
This behavior was just due to the need of implement an auxiliary class due to the class line limit of 50, changes will be kept in mind for future implementation